node/tessel-export.js

```diff
@@ -249,30 +249,40 @@ Tessel.Port = function(name, socketPath, board) {
 
   this.pwm = [];
 
-  this.I2C = function I2C(address, mode) {
-    return new Tessel.I2C({
+  this.I2C = function I2CInit(address, mode) {
+    if (port._i2c) {
+      port._i2c.disable();
+    }
+
+    port._i2c =  new Tessel.I2C({
       addr: address,
       mode: mode,
       port: port
     });
-  };
 
-  this.I2C.enabled = false;
```

Comment:
No, it wasn't redundant. `Tessel.I2C` does the following: 

- First invocation
  - Initialize an object with `addr`, `_port`, `_freq`, and `_baud` properties. 
  - If the I2C **bus** on this **port** is not yet enabled, **enable** it. 
- All following invocations
  - Initialize an object with `addr`, `_port`, `_freq`, and `_baud` properties. 

That allows > 1 I2C device to be used on a single port (up to 127), eg.

```js
var adxl345 = tessel.port.A.I2C(0x53);
var htu21d = tessel.port.A.I2C(0x40);
var mpu6050 = tessel.port.A.I2C(0x68);
```